runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -2301,22 +2422,68 @@ J9::Z::TreeEvaluator::instanceofEvaluator(TR::Node * node, TR::CodeGenerator * c
       }
    else
       {
-      return TR::TreeEvaluator::VMinstanceOfEvaluator(node, cg);
+      static bool newinstanceOf = (feGetEnv("TR_oldInstanceOf")) == NULL;
```

Comment:
We got rid of the Old evaluator in this PR, I do not think we need this bool. I am not seeing this used we can get rid of this.

Reply:
Will let the build finish and add a new commit to fix this up.

Reply:
Fixed in 85d238e